src/client/components/login/Login.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { Link } from 'react-router-dom';
+import { Route } from 'react-router-dom';
+import Authorization from "../Auth";
+import YesSession from '../YesSession';
+import LoginButtons from './LoginButtons';
+
+const UserAuth = Authorization(['Pending', 'Judge', 'Admin', 'Owner']);
+
+class Login extends React.Component {
+    render() {
+        return (
+            <div id="form">
+                <h2>Login</h2>
+                <Route path="/login" component={UserAuth(YesSession)} />
+                <LoginButtons />
+
+                <form action="/auth/login" method="POST">
+                    <input name="email" type="text" placeholder="Email" /><br />
```

Comment:
<br /> is typically a bad idea for laying out components: use styling instead (`display: block`)